src/parcsr_ls/par_mgr.c

```diff
@@ -5141,84 +5194,85 @@ hypre_MGRBuildAff( MPI_Comm comm, HYPRE_Int local_num_variables, HYPRE_Int num_f
+    return hypre_error_flag;
+  }
+  *RAP = mgr_data->RAP;
 
-   return hypre_error_flag;
+  return hypre_error_flag;
 }
 
 /* Get pointer to coarse grid solution for MGR solver */
 HYPRE_Int
 hypre_MGRGetCoarseGridSolution( void *mgr_vdata, hypre_ParVector **sol )
 {
-   hypre_ParMGRData  *mgr_data = (hypre_ParMGRData*) mgr_vdata;
+  hypre_ParMGRData  *mgr_data = (hypre_ParMGRData*) mgr_vdata;
 
-   if (!mgr_data)
-   {
-      hypre_error_in_arg(1);
-      return hypre_error_flag;
-   }
-   if (mgr_data -> U_array == NULL)
-   {
-      hypre_error_w_msg(HYPRE_ERROR_GENERIC," MGR solution array is NULL. Please make sure MGRSetup() and MGRSolve() are called \n");
-      return hypre_error_flag;
-   }
-   *sol = mgr_data->U_array[mgr_data->num_coarse_levels];
+  if (!mgr_data)
+  {
+    hypre_error_in_arg(1);
+    return hypre_error_flag;
+  }
+  if (mgr_data -> U_array == NULL)
+  {
+    hypre_error_w_msg(HYPRE_ERROR_GENERIC," MGR solution array is NULL. Please make sure MGRSetup() and MGRSolve() are called \n");
+    return hypre_error_flag;
+  }
+  *sol = mgr_data->U_array[mgr_data->num_coarse_levels];
 
-   return hypre_error_flag;
+  return hypre_error_flag;
 }
 
 /* Get pointer to coarse grid solution for MGR solver */
 HYPRE_Int
 hypre_MGRGetCoarseGridRHS( void *mgr_vdata, hypre_ParVector **rhs )
 {
-   hypre_ParMGRData  *mgr_data = (hypre_ParMGRData*) mgr_vdata;
+  hypre_ParMGRData  *mgr_data = (hypre_ParMGRData*) mgr_vdata;
 
-   if (!mgr_data)
-   {
-      hypre_error_in_arg(1);
-      return hypre_error_flag;
-   }
-   if (mgr_data -> F_array == NULL)
-   {
-      hypre_error_w_msg(HYPRE_ERROR_GENERIC," MGR RHS array is NULL. Please make sure MGRSetup() and MGRSolve() are called \n");
-      return hypre_error_flag;
-   }
-   *rhs = mgr_data->F_array[mgr_data->num_coarse_levels];
+  if (!mgr_data)
+  {
+    hypre_error_in_arg(1);
+    return hypre_error_flag;
+  }
+  if (mgr_data -> F_array == NULL)
+  {
+    hypre_error_w_msg(HYPRE_ERROR_GENERIC," MGR RHS array is NULL. Please make sure MGRSetup() and MGRSolve() are called \n");
+    return hypre_error_flag;
+  }
+  *rhs = mgr_data->F_array[mgr_data->num_coarse_levels];
 
-   return hypre_error_flag;
+  return hypre_error_flag;
 }
 
 /* Print coarse grid linear system (for debugging)*/
 HYPRE_Int
 hypre_MGRPrintCoarseSystem( void *mgr_vdata, HYPRE_Int print_flag)
 {
-   hypre_ParMGRData  *mgr_data = (hypre_ParMGRData*) mgr_vdata;
-   mgr_data->print_coarse_system = print_flag;
+  hypre_ParMGRData  *mgr_data = (hypre_ParMGRData*) mgr_vdata;
+  mgr_data->print_coarse_system = print_flag;
 
-   return hypre_error_flag;
+  return hypre_error_flag;
 }
 
 /* Print global smoother convergence factor */
 /*
 HYPRE_Int
 hypre_MGRPrintGsmoothConvergenceFactor( void *mgr_vdata, HYPRE_Int print_flag)
 {
-   hypre_ParMGRData  *mgr_data = (hypre_ParMGRData*) mgr_vdata;
-   mgr_data->print_gsmooth_convergence_factor = print_flag;
+<<<<<<< HEAD
```

Comment:
What is this? From an unfinished merge?

Reply:
Thanks for spotting this. It is inside a block that is commented out for removal. I'm gonna remove this section.